icu4j/maven-build/maven-icu4j-charset/pom.xml

```diff
@@ -0,0 +1,283 @@
+* Copyright (C) 2010-2016, International Business Machines Corporation and    *
+* others. All Rights Reserved.                                                *
+*******************************************************************************
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  
+  <parent>
+    <groupId>com.ibm.icu</groupId>
+    <artifactId>icu4j-root</artifactId>
+    <version>73.1-SNAPSHOT</version>
+    <!-- default relativePath for parent -->
+  </parent>
+  
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.ibm.icu</groupId>
+  <artifactId>icu4j-charset</artifactId>
+  <version>73.1-SNAPSHOT</version>
+
+  <packaging>jar</packaging>
+
+  <name> ICU4J Charset Provider</name>
+  <description>icu4j-charset is a supplemental library for icu4j, implementing Java Charset SPI.</description>
+  <url>https://icu.unicode.org/</url>
+  <inceptionYear>2001</inceptionYear>
+
+  <licenses>
+    <license>
+      <name>Unicode/ICU License</name>
+      <url>https://raw.githubusercontent.com/unicode-org/icu/main/icu4c/LICENSE</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <id>macchiati</id>
+      <name>Mark Davis</name>
+      <organization>Google</organization>
+      <roles>
+        <role>PMC Member</role>
+      </roles>
+    </developer>
+    <developer>
+      <id>JCEmmons</id>
+      <name>John Emmons</name>
+      <organization>IBM Corporation</organization>
+      <roles>
+        <role>PMC Member</role>
+      </roles>
+    </developer>
+    <developer>
+      <id>dougfelt</id>
+      <name>Doug Felt</name>
+      <organization>Google</organization>
+      <roles>
+        <role>PMC Member</role>
+      </roles>
+    </developer>
+    <developer>
+      <id>deborah</id>
+      <name>Deborah Goldsmith</name>
+      <organization>Apple</organization>
+      <roles>
+        <role>PMC Member</role>
+      </roles>
+    </developer>
+    <developer>
+      <id>srl295</id>
+      <name>Steven Loomis</name>
+      <organization>IBM Corporation</organization>
+      <roles>
+        <role>PMC Member</role>
+      </roles>
+    </developer>
+    <developer>
+      <id>markusicu</id>
+      <name>Markus Scherer</name>
+      <organization>Google</organization>
+      <roles>
+        <role>PMC Member</role>
+      </roles>
+    </developer>
+    <developer>
+      <id>pedberg</id>
+      <name>Peter Edberg</name>
+      <organization>Apple</organization>
+      <roles>
+        <role>PMC Member</role>
+      </roles>
+    </developer>
+    <developer>
+      <id>yumaoka</id>
+      <name>Yoshito Umaoka</name>
+      <organization>IBM Corporation</organization>
+      <roles>
+        <role>PMC Member</role>
+      </roles>
+    </developer>
+  </developers>
+
+  <mailingLists>
```

Comment:
Better to just point to https://icu.unicode.org/contacts

Reply:
this is old of course.

Reply:
@yumaoka